src/cr/cube/matrix/cubemeasure.py

```diff
@@ -248,6 +257,16 @@ def valid_overlaps(self):
         return np.sum(self._valid_overlaps[:, :, 0:2], axis=2)
 
 
+class _MrXMrOverlaps(_BaseCubeOverlaps):
+    @lazyproperty
+    def overlaps(self):
+        return self._overlaps[:, 0, :, 0]
+
+    @lazyproperty
+    def valid_overlaps(self):
+        return np.sum(self._valid_overlaps[:, 0, :, 0:2], axis=2)
+
+
```

Comment:
docstrings missed

Reply:
yeah, this was updated even prior to reading comments